test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/SigningUtilityTests.cs

```diff
@@ -15,27 +19,164 @@ namespace NuGet.Packaging.FuncTest
     [Collection(SigningTestCollection.Name)]
     public class SigningUtilityTests
     {
-        private readonly SigningTestFixture _fixture;
+        private readonly SigningTestFixture _testFixture;
+        private IList<ISignatureVerificationProvider> _trustProviders;
+        private SigningSpecifications _signingSpecifications;
 
         public SigningUtilityTests(SigningTestFixture fixture)
         {
-            if (fixture == null)
-            {
-                throw new ArgumentNullException(nameof(fixture));
-            }
-
-            _fixture = fixture;
+             _testFixture = fixture ?? throw new ArgumentNullException(nameof(fixture));
+            _trustProviders = _testFixture.TrustProviders;
+            _signingSpecifications = _testFixture.SigningSpecifications;
```

Comment:
I think you can undo the changes to lines 22-30.  `_trustProviders` and `_signingSpecifications` are unreferenced.